routes/api/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @fileOverview Main router that handles all requests to api endpoint
+ */
+const express = require('express');
+const config = require('config');
+const settings = require('./settings');
+
+const router = express.Router();
+
+router.use((req, res, next) => {
+  const key = req.query.key;
```

Comment:
If it's easy to set from the sheet, let's make this a header instead of a request param.  How about x-api-key